### ISSUE TITLE
Add Exception Handling for VAULT_PASSWORD_FILE

### ISSUE THREAD

**Issue by salmanahmed-postgresql** [stormatics/pg_cirrus#85]
As of now if the user enters wrong path for VAULT_PASSWORD_FILE pg_cirrus exits. pg_cirrus must allow wrong entries, pg_cirrus shall exit when wrong path is added for the third time for VAULT_PASSWORD_FILE.

**Comment by salmanahmed-postgresql** [comment]
Exception handling working as required, can be seen in logs below

```
Ansible vault password file: inventory.in
File 'inventory.in' does not have the correct permissions (0600).
root@xflow:~/pg_cirrus/3-node-cluster# python3 deploy.py
Welcome to pg_cirrus - Hassle-free PostgreSQL Cluster Setup


Ansible vault password file: ../pass
File '../pass' does not exist.
Ansible vault password file: ../../pass
All checks for VAULT_PASSWORD_FILE were passed
```
This issue has been fixed, pull request #94 generated against this issue. After review PR will be merged.

**Comment by salmanahmed-postgresql** [comment]
This issue has been resolved. PR #94 has been reviewed and merged.
